vignettes/Intro_to_srvyr.Rmd

```diff
@@ -0,0 +1,46 @@
+---
+title: "Intro to srvyr"
+author: "Carl Ganz"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+The `srvyr` package aims to add `dplyr` like syntax to Professor Thomas Lumley's `survey` package. In this vignette we will recreate several examples from Lumley's textbook *Complex Surveys: A Guide to Analysis Using R* using `srvyr`. 
+
+# design_survey
+
+The meat of the `sryvr` package is the `design_survey` function. This is similar to the `svydesign` function from the `survey` package, but with some `dplyr` like functionality. `design_survey` is different from `svydesign` because it supports `magrittr` pipes, it uses bare column names rather than using a "~" before the name of the variable like in the `survey` package, and you can use other `dplyr` verbs like `summarise`, `mutate`, and `group_by`. Here are some basic examples using `design_survey` and the functions `survey_total` and `survey_mean` which are the `srvyr` equivalents of `svytotal` and `svymean` respectively:

```

Comment:
Makes me wonder if I should try to fold `design_survey_rep()` into `design_survey()` -- It'd be easy to distinguish between survey designs and replicate weights based on whether the replicate argument is specified.

I hadn't because I thought it made the documentation long, but perhaps it'd be better to only have to use one function regardless of what design you're doing.

Let me know if you have an opinion @carlganz